spec/seeds/seeds_posts.sql

```diff
@@ -0,0 +1,24 @@
+-- Drop the table if it exists
+DROP TABLE IF EXISTS posts;
```

Comment:
The table can not be dropped because of the posts foreign key. You may try to use CASCADE or TRUNCATE both tables at the same time.